bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -11155,16 +11155,31 @@ public ContentTag addContentTag(Content content, TagName tagName, String comment
 	 * @throws TskCoreException
 	 */
 	public void deleteContentTag(ContentTag tag) throws TskCoreException {
+		CaseDbTransaction trans = beginTransaction();
 		acquireSingleUserCaseWriteLock();
```

Comment:
No need to get write lock with open transaction

Reply:
Ok.  Should be good to go now.